flink-formats/flink-hadoop-bulk/src/test/java/org/apache/flink/formats/hadoop/bulk/HadoopPathBasedPartFileWriterTest.java

```diff
@@ -82,13 +84,23 @@ public void testWriteFile() throws Exception {
         DataStream<String> stream =
                 env.addSource(new FiniteTestSource<>(data), TypeInformation.of(String.class));
         Configuration configuration = new Configuration();
-
+        // Elements from source  assign to one bucket , and produce two part after checkpoint.
         HadoopPathBasedBulkFormatBuilder<String, String, ?> builder =
                 new HadoopPathBasedBulkFormatBuilder<>(
                         basePath,
                         new TestHadoopPathBasedBulkWriterFactory(),
                         configuration,
-                        new DateTimeBucketAssigner<>());
+                        new BucketAssigner<String, String>() {
```

Comment:
Good idea. We could even use `BasePathBucketAssigner`. This would also make `HadoopPathBasedPartFileWriterTest#validateResult` easier because we don't have to verify that there is one bucket created. WDYT?

Reply:
Yes , I think your idea is better . I push a new commit .